src/harness/virtualFileSystemWithWatch.ts

```diff
@@ -52,80 +60,72 @@ interface Array<T> { length: number; [n: number]: T; }`
         return host;
     }
 
-    export interface File {

```

Comment:
I did my best, but I couldn't convince github's diff that I didn't add/remove `class Callbacks`. You'll have to trust me that I didn't change anything there.

TestServerHost should now be much easier to review.